### DISCUSSION TITLE
Welcome to animeflix Discussions!

### DISCUSSION THREAD

**Discussion by chirag-droid** [chirag-droid/animeflix#83] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
how to deploy it in cpanel?

**Reply by adgsenpai** [reply]
> how to deploy it in cpanel?

## Do some research on `Getting Docker to work on cPanel`

Below is a plugin I checked that enables Docker on this platform.

https://applications.cpanel.net/listings/view/Docker-for-cPanel
https://www.youtube.com/watch?v=O3Tkpy_lVAA

`Animeflix` has a _docker image_ so you can just deploy the image onto the cPanel server.

this is the docker image presently as I'm working with @chirag-droid to generate the new latest images.

https://hub.docker.com/repository/docker/chiragdroid/animeflix

you can run
`docker pull chiragdroid/animeflix`

to pull the image

#### Docker Resources

https://docs.docker.com/
https://www.youtube.com/watch?v=3c-iBn73dDE
